apps/public-docsite-v9/src/DocsComponents/Toc.stories.tsx

```diff
@@ -2,7 +2,6 @@ import * as React from 'react';
 import { addons } from '@storybook/preview-api';
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33252&commitId=e2d128cbd303f48ab3c0e331372d427963108e64&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Left bottom edge.chromium.png|3022| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic.chromium.png|376| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>